sandbox/extract-imagery-metadata_stevendepaschalis.R

```diff
@@ -87,6 +86,12 @@ extract_metadata_sd <- function(exif) {
   # Extract image file type
   image_file_format <- extract_file_type(exif)
 
+  # Check if image file format is consistent
+  if (length(unique(image_file_format)) > 1) {
+    # If not identical, return a warning
+    warning("Image file format varies across images.")
```

Comment:
This needs to go inside the function `extract_file_type`, before calculating the mode.